Enable decoder tast tests on mt8192-asurada-spherion-r0

laura-nao opened the pull request:
Enable V4L2 stateless decoder tast tests and `cros-tast-video` tast tests on `mt8192-asurada-spherion-r0` and install the required dependencies on `chromiumos-asurada`.

MT8192 decoder support is only present in the `linux-next` tree at the moment; run the tests on the `collabora-chromeos-kernel` tree for now.

laura-nao commented:
Reference results from manual LAVA jobs:

* H264: https://lava.collabora.dev/results/10922560/0_tast - score: 2/2
* VP8: https://lava.collabora.dev/results/10922577/0_tast - score:  7/7
* VP9 (group1): https://lava.collabora.dev/results/10921434/0_tast - score: 4/4
* VP9 (group2): https://lava.collabora.dev/results/10922566/0_tast - score:  2/2
* VP9 (group3): https://lava.collabora.dev/results/10922569/0_tast - score:  4/4
* VP9 (group4): https://lava.collabora.dev/results/10922572/0_tast - score:  4/4
* cros-tast-video:  https://lava.collabora.dev/results/10932951/0_tast - score: 20/21

gctucker commented:
Adding skip label because of #1994

nuclearcat commented:
Not sure why, but tests failed to run: https://storage.chromeos.kernelci.org/kernelci/chromeos-stable/chromeos-stable-20230801.0/arm64/cros---chromeos-6.1-arm64-chromiumos-mediatek.flavour.config+arm64-chromebook+CONFIG_MODULE_COMPRESS_GZIP=n/clang-14/lab-collabora/cros-tast-video-mt8192-asurada-spherion-r0_chromeos.html
Also can't see anything yet on new runs for -stateless-* tests.

laura-nao commented:
I rebased the branch on top of the latest `chromeos` and updated the list of tast tests according to the version of ChromeOS currently flashed (R114). 
Reference manual LAVA jobs with the latest `collabora-chromeos-kernel` build:
- cros-tast-mm-decode-v4l2-stateless-h264: https://lava.collabora.dev/results/11324922
- cros-tast-mm-decode-v4l2-stateless-vp8: https://lava.collabora.dev/results/11324922
- cros-tast-mm-decode-v4l2-stateless-vp9-group1: https://lava.collabora.dev/results/11325063
- cros-tast-mm-decode-v4l2-stateless-vp9-group2: https://lava.collabora.dev/results/11325064
- cros-tast-mm-decode-v4l2-stateless-vp9-group3: https://lava.collabora.dev/results/11325065
- cros-tast-mm-decode-v4l2-stateless-vp9-group4: https://lava.collabora.dev/results/11325066
- cros-tast-mm-decode: https://lava.collabora.dev/results/11325080

I can see the tast decoder test jobs being correctly listed by `kci_test` when using bmeta.json and artifacts.json from the latest `collabora-chromeos-kernel` build (https://storage.chromeos.kernelci.org/collabora-chromeos-kernel/for-kernelci/v6.5-rc3-5446-g3f88094ab961/arm64/defconfig%2Barm64-chromebook/gcc-10/).

nuclearcat commented:
I think we can merge it, since it is verified and proven to work manually and expected to run only on collabora-chromeos-kernel, which is updated too rarely, to wait results on dashboard.
